services/web/server/src/simcore_service_webserver/email_handlers.py

```diff
@@ -0,0 +1,116 @@
+import logging
+from typing import Any, Literal, Optional
+
+from aiohttp import web
+from pydantic import BaseModel, EmailStr, Field
+from servicelib.aiohttp.requests_validation import parse_request_body_as
+
+from ._meta import API_VTAG
+from .email_core import (
+    check_email_server_responsiveness,
+    get_plugin_settings,
+    send_email_from_template,
+)
+from .login.decorators import login_required
+from .products import Product, get_current_product, get_product_template_path
+from .security_decorators import permission_required
+from .utils import get_traceback_string
+from .utils_aiohttp import envelope_json_response
+
+logger = logging.getLogger(__name__)
+
+
+#
+# API schema models
+#
+
+
+class TestEmail(BaseModel):
+    from_: Optional[EmailStr] = Field(None, description="Email sender")
```

Comment:
why is the field called `from_`?

Reply:
``from`` is a reserved word e.g. ``from pydantic import *``